### PR TITLE
Update/Reminder mail notification for enrolled teachers

### PR DESCRIPTION
**Pull request by nayanasQA**
@meyyappanv @Naveen1895 Request you to review the code.

### CODE REVIEW

**Comment by Naveen1895** [comment]
@nayanalinkstreet From the next PR onwards, Could you follow branch naming conventions ` <type of task>/<short-desciption-of-task-separated-by-hypens>`? :) :)

File: locallib.php [hunk 1]
```diff
@@ -612,33 +620,35 @@ function zoom_get_mail_body($event, $attendee, &$param) {
...
+    	return $DB->get_records_sql("SELECT DISTINCT u.id, CONCAT(u.firstname ,' ', u.lastname) AS fullname, u.timezone
                                    FROM {user} u
                                    JOIN {role_assignments} ra ON ra.userid = u.id
                                   WHERE u.deleted = 0
                                         AND u.suspended = 0
-                                        AND ra.roleid = 5
+                                        AND ra.roleid IN ({$recipient_roles})
                                         AND contextid = ?", array(context_course::instance($courseid)->id)
-    );
+    									);
```
Comment (Naveen1895) [inline locallib.php]: @nayanalinkstreet Could you format this ?

File: mod_form.php [hunk 2]
```diff
@@ -323,6 +323,11 @@ public function definition() {
         ), null , get_string('auto_recording', 'zoom'));
         $mform->setDefault('auto_recording', $config->defaultautorecording);
 
+        //Enable mail notifications for teachers
```
Comment (Naveen1895) [inline mod_form.php]: @nayanalinkstreet I feel this will be good if you bring the whole option below the remainder mail checkbox

**Review by Naveen1895 — CHANGES_REQUESTED** [review]

**Comment by Naveen1895** [comment]
@nayanalinkstreet Can you increase the zoom plugin version?

File: version.php [hunk 3]
```diff
@@ -25,7 +25,7 @@
 defined('MOODLE_INTERNAL') || die();
 
 $plugin->component = 'mod_zoom';
-$plugin->version = 2020081800;
+$plugin->version = 2020091000;
 $plugin->release = 'v2.4.1';
```
Comment (Naveen1895) [inline version.php]: @nayanalinkstreet Could you make it as 2.5.0?

**Review by Naveen1895 — APPROVED** [review]

**Review by meyyappanv — APPROVED** [review]

**Pull request merged**